### PR TITLE
specify generated shapefile projection

### PR DESCRIPTION
**Pull request by dunn**

### CODE REVIEW

**Comment by dunn** [comment]
test failures all look like flaky http specs

**Comment by dunn** [comment]
green now

File: config/locales/geoblacklight.en.yml [hunk 1]
```diff
@@ -15,7 +15,7 @@ en:
       error_with_url: 'Sorry, the requested file could not be downloaded. Try downloading it directly from: %{link}'
       export_formats: 'Export Formats'
       export_link: 'Export'
-      export_shapefile_link: 'Shapefile'
+      export_shapefile_link: 'WGS 84 Shapefile'
```
Comment (mejackreed) [inline config/locales/geoblacklight.en.yml:18]: Can we guarantee that this is a WGS 84 shapefile for all metadata?

Comment (dunn) [reply config/locales/geoblacklight.en.yml:18]: Currently, that's how data is reprojected, when it's fetched from GeoServer.  I'm not sure if that can be changed, though.  cc @tbrittnacher

**Pull request closed without merging**